### ISSUE TITLE
just want to say the title BREAKING Update was epic 

### ISSUE THREAD

**Issue by jonnytracker** [kijai/ComfyUI-CogVideoXWrapper#265]
its a big update yet breaking .. haha got me there, yo epic bro

**Comment by jav12z** [comment]
I don't know about the rest, but I have been forced to return to the previous version because all my videos have lost a lot of quality.

The videos from img2video using an initial and a final frame for cogvideoxfun 2B 1.1 tend to have more noise and glitches as they approach the final frame. The same happens with version 5B. Fortunately, the Reward Loras work quite well with the older versions of the custom node.

**Comment by kijai** [comment]
> I don't know about the rest, but I have been forced to return to the previous version because all my videos have lost a lot of quality.
> 
> The videos from img2video using an initial and a final frame for cogvideoxfun 2B 1.1 tend to have more noise and glitches as they approach the final frame. The same happens with version 5B. Fortunately, the Reward Loras work quite well with the older versions of the custom node.

Can you post example of this? As in same video with same settings from both versions. I don't use 2b at all so could've missed something. 5b results haven't changed at least.

**Comment by jonnytracker** [comment]
> I don't know about the rest, but I have been forced to return to the previous version because all my videos have lost a lot of quality.
> 
> The videos from img2video using an initial and a final frame for cogvideoxfun 2B 1.1 tend to have more noise and glitches as they approach the final frame. The same happens with version 5B. Fortunately, the Reward Loras work quite well with the older versions of the custom node.

im back to 1.1  it understands the prompt more.. 1.5 seems too stupid to understand any prompt

**Comment by zazoum-art** [comment]
Is there any additional change on the 1.5 kijai model between the test branch and the update8 one. Can the sampler change between the two versions affect what the model understands and what not?

**Comment by zazoum-art** [comment]
Well for me the kijai 1.5 model still works fine, in a challenging long 185 frames footage for job (not for films hobby), the flux input female player and her skirt and ponytail and general movement follow my prompt as I want... except for the damn racket net that gets deformed all the time... I guess I have to train tennis lora for the net. I don't know if I could fix it with more tokens, but I run out of them.
Long story short, the kijai 1.5 understands the prompt very well as before the update.

**Comment by jav12z** [comment]
> > I don't know about the rest, but I have been forced to return to the previous version because all my videos have lost a lot of quality.
> > The videos from img2video using an initial and a final frame for cogvideoxfun 2B 1.1 tend to have more noise and glitches as they approach the final frame. The same happens with version 5B. Fortunately, the Reward Loras work quite well with the older versions of the custom node.
> 
> Can you post example of this? As in same video with same settings from both versions. I don't use 2b at all so could've missed something. 5b results haven't changed at least.

Hi, I've reverted my ComfyUI installation to ComfyUI v0.2.7, and now everything seems to go well.
